<launch>
    <param name="use_sim_time" value="true"/>

    <!-- set namespace -->
    <group ns="simple_leg">
        <arg name="model" default="simple_leg.xacro"/>
        <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find choreonoid_ros_control_sample)/xacro/$(arg model)'"/>

        <!-- robot_state_publisher -->
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="true"/>

        <!-- load controller from rosparam -->
        <rosparam file="$(find choreonoid_ros_control_sample)/config/simple_leg_control.yaml" command="load"/>
        <node name="controller_spawner" pkg="controller_manager"
            type="spawner" output="screen" respawn="true"
            args="joint_state_controller
                Hip_position_controller
                Knee_position_controller"/>
    </group>
</launch>
